solaris/packer/scripts/solaris/postinstall.sh

```diff
@@ -15,13 +20,62 @@ pkgutil -y -i libtool
 pkgutil -y -i wget
 pkgutil -y -i curl
 pkgutil -y -i gcc5core
+pkgutil -y -i gcc5g++
 pkgutil -y -i perl
 pkgutil -y -i sudo
 pkgutil -y -i git
+pkgutil -y -i nano
 rm /usr/bin/perl
 mv /opt/csw/bin/perl5.10.1 /usr/bin/
 mv /usr/bin/perl5.10.1 /usr/bin/perl
 
+# Compile GCC-5.5 and CMake
+curl -L http://packages.wazuh.com/utils/gcc/gcc-5.5.0.tar.gz | gtar xz
+cd gcc-5.5.0
+curl -L http://packages.wazuh.com/utils/gcc/mpfr-2.4.2.tar.bz2 | gtar xj
+mv mpfr-2.4.2 mpfr
+curl -L http://packages.wazuh.com/utils/gcc/gmp-4.3.2.tar.bz2 | gtar xj
+mv gmp-4.3.2 gmp
+curl -L http://packages.wazuh.com/utils/gcc/mpc-0.8.1.tar.gz | gtar xz
+mv mpc-0.8.1 mpc
+curl -L http://packages.wazuh.com/utils/gcc/isl-0.14.tar.bz2 | gtar xj
+mv isl-0.14 isl
+cd ..
+unset CPLUS_INCLUDE_PATH
+unset LD_LIBRARY_PATH
+mkdir -p gcc-build
+cd gcc-build
+# Fix for solaris 10/11
+if [ "$(uname -v)" = "11.3" ]; then
+  ../gcc-5.5.0/configure --prefix=/usr/local/gcc-5.5.0 --enable-languages=c,c++ --disable-multilib --disable-libsanitizer --disable-bootstrap --with-ld=/usr/ccs/bin/ld --without-gnu-ld --with-gnu-as --with-as=/opt/csw/bin/gas
+  gmake && gmake install
+  echo "export PATH=/usr/local/gcc-5.5.0/bin:/usr/local/bin:/opt/csw/bin:${PATH}" >> /etc/profile
+  export PATH="/usr/local/gcc-5.5.0/bin:/usr/local/bin:/opt/csw/bin:${PATH}"
+  export CPLUS_INCLUDE_PATH=/usr/local/gcc-5.5.0/include/c++/5.5.0
+  export LD_LIBRARY_PATH=/usr/local/gcc-5.5.0/lib
+else
+  ../gcc-5.5.0/configure --prefix=/usr/local/gcc-5.5.0 --enable-languages=c,c++ --disable-multilib --disable-libsanitizer --disable-bootstrap --with-gnu-as --with-as=/usr/sfw/bin/gas
+  gmake && gmake install
+  echo "export PATH=/usr/local/gcc-5.5.0/bin:${PATH}" >> /etc/profile
+  PATH="/usr/local/gcc-5.5.0/bin:${PATH}"
+  export PATH
+  CPLUS_INCLUDE_PATH=/usr/local/gcc-5.5.0/include/c++/5.5.0/
+  export CPLUS_INCLUDE_PATH
+  LD_LIBRARY_PATH=/usr/local/gcc-5.5.0/lib/
+  export LD_LIBRARY_PATH
+fi
+
+echo "export CPLUS_INCLUDE_PATH=/usr/local/gcc-5.5.0/include/c++/5.5.0/" >> /etc/profile
+echo "export LD_LIBRARY_PATH=/usr/local/gcc-5.5.0/lib/" >> /etc/profile
+cd .. && rm -rf gcc-*
```

Comment:
```suggestion
rm -rf gcc-*
```

Reply:
Disagree, that cd is required: https://github.com/wazuh/wazuh-packages/blob/2da6ce8f8a9ef51428fe5303a43e15f6c055e2e5/solaris/packer/scripts/solaris/postinstall.sh#L47

Reply:
29423b610bba9bf79daff729fd4319ec7341c455